docs/UserGuide.md

```diff
@@ -1,178 +1,158 @@
+  
+## Table of Contents  
+- [Quick Start](#quick-start)   
+- [Features](#features)  
+    * [Creating a budget book: `create`](#creating-a-budget-book-create)
+    * [Opening a budget book: `open`](#opening-a-budget-book-open)
+    * [Closing a budget book: `close`](#closing-a-budget-book-close)
+    * [Adding an expenditure: `add`](#adding-an-expenditure-add)
+    * [Listing all budget books or expenditures: `list`](#listing-all-budget-books-or-expenditures-list)
+    * [Locating expenditures by name: `find`](#locating-expenditures-by-name-find)
+    * [Deleting a budget book or expenditure: `del`](#deleting-a-budget-book-or-expenditure-del)
+    * [Listing available commands: `help`](#listing-available-commands-help)
+    * [Exiting the program: `exit`](#exiting-the-program-exit)
+- [Command Summary](#command-summary)  
+  
+  
+## Quick Start  
+  
+1. Ensure you have Java `11` or above installed in your Computer.  
+      
+2. Download the latest `NUSave.jar` from [here](https://github.com/AY2021S1-CS2103T-T11-4/tp/releases).  
+      
+3. Copy the file to the folder you want to use as the *home folder* for your NUSave.  
+      
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds.  
+      
+5. Type the command in the command box and press enter to execute it. Some example commands you can try:  
+    - `list`: Lists all budget books.  
+      
+   - `create Temasek Hall`: Adds a budget book named Temasek Hall to NUSave.  
+      
+   - `open 1`: Opens the 3rd budget book (Temasek Hall) shown in the current list.  
+      
+   - `add  Hall Fees 2600`: Adds an expenditure named `Hall Fees` of price $2600 to the budget book (Temasek Hall) that is open.  
+      
+   - `del 1`: Deletes the 1st expenditure (Hall Fees) in the list of expenditures.  
+      
+   - `close`: Closes the budget book (Temasek Hall) that is open.  
+      
+   - `del 1`: Deletes the 1st budget book (Temasek Hall) in the list of budget books.  
+      
+   - `help`: Shows the list of commands available for NUSave.  
+      
+   - `exit` : Exits the app.  
+     
+6. Refer to the [features](#features) below for details of each command.  
+  
+---  
+## Features  
+  
+### Creating a budget book: `create`
+  
+Creates a budget book in NUSave.  
+  
+Format: `create NAME`  
+  
+- This command is only available when no budget book is open.  
+- Creates a budget book with the given `NAME`.  
+     
+Examples:  
+- `create Daily Expenses`  
+- `create School Related Expenses`  
+    
+    
+### Opening a budget book: `open`
+  
+Opens a budget book in NUSave.  
+  
+Format: `open INDEX`  
+- This command is only available when no budget book is open.  
+- Opens the budget book at the specific `INDEX`.      
+- `INDEX` refers to the index number shown in the displayed budget book list.  
+- The index must be a positive integer e.g. 1, 2, 3, ...  
```

Comment:
Will it be better if "must be a positive integer" is bolded to draw users' attention to this restriction?